src/Simulation/Algorithms/COE/RV2COE.hpp

```
@@ -0,0 +1,196 @@
+            arglat = acos(cos_arglat);
+            
+            if (r.z < 0.0)
+                arglat = TWOPI - arglat;
+
+            m = arglat;
+        }
+
+    
+        // Longitude of periapsis (non-circular, equatorial orbit)
+        double lonper = NAN;
+        if (e_mag > R_EPSILON && orbitGeom != CIRCULAR && orbitIncl == EQUATORIAL) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)